lib/count_sim_swaps/main.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+from datetime import datetime, timezone
+import os
+from pngme.api import AsyncClient
+
+
+async def get_count_of_sim_swaps(
+    api_client: AsyncClient,
+    user_uuid: str,
+    utc_timestamp: datetime,
+    semi_window_days: int,
```

Comment:
How about we use `utc_endtime` so we maintain the same method signatures as other features?

Reply:
The issue is that this is not an end time, it is a "window central time". It is not that I am in love with `uct_timestamp` but end time does not fit. Another suggestion?

Reply:
Gotcha 👍